control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -79,6 +79,8 @@ var (
 	// only once.
 	deleteDNSOperatorDeploymentOnce sync.Once
 	deleteCVORemovedResourcesOnce   sync.Once
+
+	DeletionSecondsTimeout = 240 * time.Second
```

Comment:
I would give resources more time, given that some can take a while to get destroyed, 10min would be better.

Reply:
nit: this doesn't look like it needs to be a public var (can start with lower case)